pyshtools/_version.py

```diff
@@ -5,8 +5,9 @@
 # directories (produced by setup.py build) will contain a much shorter file
```

Comment:
I don't think that versioneer works properly without a setup.py file. I will look into using setuptools scm instead.